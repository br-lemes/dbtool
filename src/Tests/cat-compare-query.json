[1m[93m/home/breno/src/dbtool/.b.json[39m[0m[2m --- JSON[0m
[2m 1 [0m[                                  [2m 1 [0m[
[91;1m 2 [0m    [91;1m{[0m                              [2m . [0m
[91;1m 3 [0m        [91m"id"[0m[91m:[0m [91m2[0m[91m,[0m                   [2m . [0m
[91;1m 4 [0m        [91m"title"[0m[91m:[0m [91m"Another Post"[0m[91m,[0m   [2m . [0m
[91;1m 5 [0m        [91m"name"[0m[91m:[0m [91m"John Doe"[0m         [2m . [0m
[91;1m 6 [0m    [91;1m}[0m[91m,[0m                             [2m . [0m
[2m 7 [0m    {                              [2m 2 [0m    {
[2m 8 [0m        [95m"id"[0m: 1,                   [2m 3 [0m        [95m"id"[0m: 1,
[2m 9 [0m        [95m"title"[0m: [95m"My First Post"[0m,  [2m 4 [0m        [95m"title"[0m: [95m"My First Post"[0m,
[2m10 [0m        [95m"name"[0m: [95m"John Doe"[0m         [2m 5 [0m        [95m"name"[0m: [95m"John Doe"[0m
[2m11 [0m    },                             [2m 6 [0m    },
[2m.. [0m                                   [92;1m 7 [0m    [92;1m{[0m
[2m.. [0m                                   [92;1m 8 [0m        [92m"id"[0m[92m:[0m [92m2[0m[92m,[0m
[2m.. [0m                                   [92;1m 9 [0m        [92m"title"[0m[92m:[0m [92m"Another Post"[0m[92m,[0m
[2m.. [0m                                   [92;1m10 [0m        [92m"name"[0m[92m:[0m [92m"John Doe"[0m
[2m.. [0m                                   [92;1m11 [0m    [92;1m}[0m[92m,[0m
[2m12 [0m    {                              [2m12 [0m    {
[2m13 [0m        [95m"id"[0m: 3,                   [2m13 [0m        [95m"id"[0m: 3,
[2m14 [0m        [95m"title"[0m: [95m"Jane's Blog"[0m,    [2m14 [0m        [95m"title"[0m: [95m"Jane's Blog"[0m,

