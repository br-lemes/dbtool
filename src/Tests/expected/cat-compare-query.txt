/home/breno/src/dbtool/.b.json --- JSON
No changes.

